config/packages/prod/monolog.yaml

```diff
@@ -0,0 +1,16 @@
+monolog:
+    handlers:
+        main:
+            type: fingers_crossed
+            action_level: error
+            handler: nested
+            excluded_http_codes: [404, 405]
+            buffer_size: 50 # How many messages should be saved? Prevent memory leaks
```

Comment:
Yes I know, I think all possible features have to be verified in production mode. Certainly because there is not a lot of end-to-end coverage of all the features that should be supported. When all changes land in deploy I want to test all functionality on test2 and will reserve some time to do so.